bar_app/tests/reference/test_reference_model.py

```diff
@@ -0,0 +1,81 @@
+from django.test import TestCase
+from bar_app.models import Reference
+
+class ReferenceModelTests(TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        Reference.objects.create(ref="REF001", name="name 1", description="description 1")
+        Reference.objects.create(ref="REF002", name="name 2")
```

Comment:
you should define those as class attributes and reuse them in all your tests. you would save at least one line of code in each test
(like you are doing in test_reference_view)